src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10802,7 +10824,7 @@ void func_80848C74(PlayState* play, Player* this) {
     }
 }
 
-void func_80848EF8(Player* this) {
+void Player_UpdateStoneOfAgony(Player* this) {
```

Comment:
Just curious, have you looked at the docs for this function in MM? It'd identical to the OoT function except it doesn't check `QUEST_STONE_OF_AGONY`. It's called `Player_DetectSecrets` in MM which is named on what this function is doing instead of tying it to the stone of agony.

Reply:
I see. I dont really like the "secrets" terminology in general personally, if I read that for the first time I would not be able to deduce that its for stone of agony. But I do understand the desire for cross game sync

Reply:
what is this doing in mm anyway? Does mm rumble for similar things as stone of agony, but you just dont need the in-game item for it to work?

Reply:
If that is the case, id prefer "secrets" in all cases be "rumble secrets" at least, so that its obvious what its referring to

Reply:
Fwiw, I would say `Player_DetectRumbleSecrets` would be fine for both games.

Reply:
coming back to this-- I think I would like to keep the pattern of naming "update" type functions called directly by `_Update` as `UpdateX`
So, I think `Player_UpdateRumbleSecrets` could work. Its both updating the weird accumulator variable for the secrets and actually doing the rumbling

Reply:
After even more thought on this, Ill go with `Player_DetectRumbleSecrets`, I think its okay to not have "update" in this one